docs/src/content/docs/commands/SELECT.md

```diff
@@ -3,31 +3,39 @@ title: SELECT
 description: Documentation for the DiceDB command SELECT
 ---
 
+**Note:** As of today, DiceDB no longer supports multiple databases. Therefore, the `SELECT` command is now a dummy method and does not affect the database. It remains as a placeholder.
```

Comment:
I believe we can frame it as below, so that it doesn't come across we earlier supported multiple databases.
> As of today, DiceDB does not support multiple databases. Therefore, the `SELECT` command is currently a dummy method and does not affect the database. It remains as a placeholder.